### ISSUE TITLE
[Feature Proposal] Löschen auch nach VF übertragen

### ISSUE THREAD

**Issue by julianschick** [Akaflieg/startkladde#9]
Beim Löschen eines Fluges, der nach VF übertragen wurde, könnte man einen Tabelleneintrag erstellen in einer eigenen Tabelle, die dann beim Upload abgearbeitet wird. Was hältst du davon @eehmke?

**Comment by eehmke** [comment]
Vielleicht braucht man keine extra Tabelle. Ein Eintrag "deleted" im Flight 
Datensatz dürfte reichen. Dann ist der Eintrag gleich über die Datenbank 
persistent, zusammen mit vfid und uploaded.

Beim Löschen von Flügen wird dann nicht der Flug gelöscht, sondern das Flag 
deleted auf true gesetzt.

Beim Übertragen zum VF werden gelöschte Flüge auf dem VF gelöscht und dann in 
der SK-Datenbank gelöscht.

In der Flugliste werden gelöschte Flüge nicht angezeigt.

Ist das ein Konzept? Das würde sogar die Möglichkeit eröffnen, versehentlich 
gelöschte Flüge in der SK wieder herzustellen.`

Am Dienstag, 29. November 2022, 11:05:43 CET schrieb Julian Schick:

**Comment by julianschick** [comment]
Ja, so hatte ich mir das gedacht. Ob man nun gelöschte Flüge behalten und flaggen sollte, weiß ich nicht. Das wird zumindest ziemlich aufwändig, weil man an allen Stellen gucken muss, dass die wirklich nicht dabei sind. Bei Statistiken, Überlappungschecks, etc. pp.

Aber einfach nur die VF-ID behalten und als Löschauftrag ablegen, das geht auf jeden Fall.
